### DISCUSSION TITLE
Welcome to simple-php-skeleton Discussions!

### DISCUSSION THREAD

**Discussion by Pixelao** [Pixelao/simple-php-skeleton#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pixelao** [comment]
Test

**Comment by diegomarty** [comment]
It is a good skeleton of php, I will use it in future projects, thanks for your contribution. 👌👌

**Reply by Pixelao** [reply]
I know bitch
